src/libcpu/src/jit/jit.cpp

```diff
@@ -265,16 +379,25 @@ gen(JitBlock &block)
          a.bind(targetIter->second.label);
       }
 
-      if (JIT_DEBUG) {
```

Comment:
This is needed for the debugger to function properly when access violations occur.